tests/cypress/integration/canvas3d_functionality/case_78_canvas3d_functionality_cuboid_label.js

```diff
@@ -0,0 +1,180 @@
+            );
+        });
+
+        it('Hide/unhide a cuboid via sidear. The cuboid on the perpective/top/side/front view be hidden/unhidden.', () => {
+            cy.get('#cvat-objects-sidebar-state-item-1')
+                .find('.cvat-object-item-button-hidden')
+                .click(); // Hide the cuboid
+            cy.get('.cvat-object-item-button-hidden-enabled').should('exist').trigger('mouseout');
+            cy.get('.cvat-canvas3d-perspective').screenshot('canvas3d_perspective_hide_cuboid');
+            cy.compareImagesAndCheckResult(
+                `${screenshotsPath}/canvas3d_perspective_disable_occlud_cuboid.png`,
+                `${screenshotsPath}/canvas3d_perspective_hide_cuboid.png`,
+            );
+            ['topview', 'sideview', 'frontview'].forEach((view) => {
+                cy.get(`.cvat-canvas3d-${view}`)
+                    .find('.cvat-canvas3d-fullsize')
+                    .screenshot(`canvas3d_${view}_hide_cuboid`);
+            });
+            [
+                ['canvas3d_topview_unlock_cuboid.png', 'canvas3d_topview_hide_cuboid.png'],
+                ['canvas3d_sideview_unlock_cuboid.png', 'canvas3d_sideview_hide_cuboid.png'],
+                ['canvas3d_frontview_unlock_cuboid.png', 'canvas3d_frontview_hide_cuboid.png'],
+            ].forEach(([viewAfterCubiodUnlock, viewAfterCubiodHide]) => {
+                cy.compareImagesAndCheckResult(`${screenshotsPath}/${viewAfterCubiodUnlock}`, `${screenshotsPath}/${viewAfterCubiodHide}`);
+            });
+            cy.get('.cvat-object-item-button-hidden-enabled').click(); // Unhide the cuboid
+            cy.get('.cvat-object-item-button-hidden').should('exist').trigger('mouseout');
+            cy.get('.cvat-canvas3d-perspective').screenshot('canvas3d_perspective_unhide_cuboid');
+            cy.compareImagesAndCheckResult(
+                `${screenshotsPath}/canvas3d_perspective_hide_cuboid.png`,
+                `${screenshotsPath}/canvas3d_perspective_unhide_cuboid.png`,
+            );
+            ['topview', 'sideview', 'frontview'].forEach((view) => {
+                cy.get(`.cvat-canvas3d-${view}`)
+                    .find('.cvat-canvas3d-fullsize')
+                    .screenshot(`canvas3d_${view}_unhide_cuboid`);
+            });
+            [
+                ['canvas3d_topview_hide_cuboid.png', 'canvas3d_topview_unhide_cuboid.png'],
+                ['canvas3d_sideview_hide_cuboid.png', 'canvas3d_sideview_unhide_cuboid.png'],
+                ['canvas3d_frontview_hide_cuboid.png', 'canvas3d_frontview_unhide_cuboid.png'],
+            ].forEach(([viewAfterCubiodHide, viewAfterCubiodUnhide]) => {
+                cy.compareImagesAndCheckResult(`${screenshotsPath}/${viewAfterCubiodHide}`, `${screenshotsPath}/${viewAfterCubiodUnhide}`);
+            });
+        });
+
+        it.skip('Switch pinned property for a cuboid via sidear. The cuboid can be resized but cannot be moved.', () => {
+            // TODO
+        });
```

Comment:
Do we need this empty test in this patch?

Reply:
Removed.